editor/sidebar/post-author/index.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+import { PanelRow, withInstanceId } from 'components';
+import { Component } from 'element';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getEditedPostAttribute } from '../../selectors';
+import { editPost } from '../../actions';
+
+class PostAuthor extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			authors: [],
+		};
+	}
+
+	fetchAuthors() {
+		this.fetchAuthorsRequest = new wp.api.collections.Users().fetch( { data: {
+			roles: 'author,editor,administrator',
```

Comment:
Do we need to define `roles` here? Two issues

- This doesn't include `contributors` (easily spotted for me because I have a site with two users, one of which a contributor 😛 )
- This won't respect custom roles: https://developer.wordpress.org/plugins/users/roles-and-capabilities/#adding-roles